components/list/test/list.visual-diff.js

```diff
@@ -147,6 +147,18 @@ describe('d2l-list', () => {
 			{ name: 'skeleton', selector: '#selectableSkeleton' },
 			{ name: 'extended separators', selector: '#selectableSeparatorsExtended' }
 		] },
+		{ category: 'selectableHref', tests: [
+			{ name: 'hover href', selector: '#selectableHref', action: () => hover('#selectableHref [selectable]') },
+			{ name: 'hover selection', selector: '#selectableHref', action: async() => {
+				const [x, y, height] = await page.$eval('#selectableHref [selectable]', elem => {
+					const rect = elem.getBoundingClientRect();
+					return [rect.x, rect.y, rect.height];
+				});
+				page.mouse.move(x + 5, y + height / 2);
+			} },
```

Comment:
**hover selection** test hovers over the area of the `list-item` that is not primary action, hence activating selection styles.

Reply:
Is it possible to grab a specific element here instead of manually moving the mouse? I'm a bit worried this could be finicky and might break if we change the component.

Reply:
I tried that, shadow dom seems to prevent selecting items that are generated and not directly declared as as child. Ideally `[slot="control"]` inside the element would be selected.

Reply:
Could you instead just manually add the `_hovering-selection` attribute to the list-item?